plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -727,12 +729,185 @@ public void testTimestamp(ZoneId sessionZone)
                 .addRoundTrip("timestamp", timestampDataType(3).toLiteral(timeGapInKathmandu), TIMESTAMP_MILLIS, timestampDataType(3).toLiteral(timeGapInKathmandu))
                 // max value in Oracle
                 .addRoundTrip("timestamp", "TIMESTAMP '9999-12-31 00:00:00.000'", TIMESTAMP_MILLIS, "TIMESTAMP '9999-12-31 00:00:00.000'")
-                .execute(getQueryRunner(), session, oracleCreateAndInsert("test_timestamp"))
+                // From docs: In Oracle, fractional_seconds_precision is optional and specifies the number of digits in the fractional part of the SECOND datetime field. It can be a number (0-9) with the default being 6.
+                // So default precision will timestamp(6)
+                //.execute(getQueryRunner(), session, oracleCreateAndInsert("test_timestamp"))
```

Comment:
comment `//.execute(getQueryRunner(), session, oracleCreateAndInsert("test_timestamp"))` is required?

Reply:
Previously it was there, because we supported only timestamp(3).
But because now we support any precision if we create just timestamp - default precision became 6.
So I comment this line and added more tests.
I think I can delete it - it was left more for reviewers, to explain why this line is deleted.

Reply:
refactored

Reply:
refactored